faq.rst

```diff
@@ -112,6 +112,46 @@ explicitly with ::
 
 https://en.wikipedia.org/wiki/EGL_(API)
 
+How to achieve transparency with 2D objects?
+--------------------------------------------
+
+It is possible to render 2D visuals, such as ``ImageVisual``, with translucency
+(i.e. partial transparency, a see-through effect). Here are the key points to
+achieve this, using ``ImageVisual`` as an example:
+
+1. Set the opcity value in the alpha channel of the images: ::
+  
+    # A white image with integer values between 0 and 255.
+    image_data1 = np.ones((200, 300, 4), dtype='uint8')
+    # Half translucent.
+    image_data1[..., 3] = 128
+    # A blue image with float values between 0 and 1.
+    image_data2 = np.zeros((200, 300, 4), dtype='np.float32')
+    image_data2[..., 2]  = 1.0  # Blue.
+    # A bit more translucent.
+    image_data2[..., 3] = 0.25
+    visual1 = ImageVisual(image_data1)
+    visual2 = ImageVisual(image_data2)
+
+2. Enable translucency for the visuals: ::
+
+    # This is the default for ImageVisual, so it could be skipped, but it could
+    # be needed with other types of visuals.
+    visual1.set_gl_state('translucent')
+    visual2.set_gl_state('translucent')
```

Comment:
I think this can be removed. It is the default for a reason. I think it could be mentioned at the end that some visuals may not work properly because of their default GL settings. These settings can be configured by using this function. However, while this would be good to mention, I think VisPy is missing some important documentation about what these presets map to for individual settings and even what the individual settings mean.

Reply:
Sounds good. Putting it at the end.

For the [presets](https://github.com/vispy/vispy/blob/df09b3079d71b7d1942481a051b900f145c4cfee/vispy/gloo/wrappers.py#L32), do you know if could document them from the code, like we have docstrings on functions? That would be a first step, for the api doc.

Reply:
I'm not sure how API docs work for "constants" like the presets dictionary. It could be done in the `set_gl_state` function or one of the other ones that deals with these presets. I think `set_gl_state` on the Visual is the most common (I think its on the Visual).

Reply:
That seems like a good idea.